src/System.Net.Primitives/src/System/Net/IPAddress.cs

```diff
@@ -614,7 +615,13 @@ public override int GetHashCode()
             {
                 if (_hashCode == 0)
                 {
```

Comment:
comment above still refers to generating hash from string repr'n

Reply:
Thanks, I updated it to match the new implementation.